packages/common/container-definitions/api-report/container-definitions.legacy.alpha.api.md

```diff
@@ -204,17 +204,13 @@ export interface IDeltaManager<T, U> extends IEventProvider<IDeltaManagerEvents>
     readonly active: boolean;
     readonly clientDetails: IClientDetails;
     readonly hasCheckpointSequenceNumber: boolean;
-    // @deprecated
-    readonly inbound: IDeltaQueue<T>;
     readonly inboundSignal: IDeltaQueue<ISignalMessage>;
```

Comment:
It's really weird that we are keeping inboundSignal here. Can we remove it as part of this change?
If not, can we mark it deprecated and open a task item to remove it in the future?

Reply:
We will follow up with a separate task for that.